docs/source/ja/_toctree.yml

```
@@ -4,11 +4,32 @@
   - local: installation
     title: インストール
   title: はじめに
+- sections:
+  - local: quicktour
+    title: クイックツアー
 - sections:
   - local: accelerate
     title: 🤗 Accelerate を用いた分散学習
   title: チュートリアル
 - sections:
   - sections:
     - local: multilingual
-      title: 推論のための多言語モデル
\ No newline at end of file
+      title: 推論のための多言語モデル
+- sections:
+  - local: autoclass_tutorial
+    title: AutoClassを使用してポータブルなコードを書く
+- sections:
+  - local: preprocessing
+    title: データ前処理
+- sections:
+  - local: training
+    title: 訓練
+- sections:
+  - local: model_sharing
+    title: モデル共有
+- sections:
+  -local: pipeline_tutorial
+   title: パイプライン共有
+- sections:
+  -local: add_new_model
+   title: 新しいモデルを追加する
```

I think the formatting here might've been giving the doc-builder some hiccups

```suggestion
  - local: pipeline_tutorial
     title: パイプライン共有
- sections:
  - local: add_new_model
     title: 新しいモデルを追加する
```